docs/admin/runai-setup/authentication/sso.md

```diff
@@ -41,34 +41,45 @@ You can configure your IdP to map several IdP attributes:
 
 ## Step 1: UI Configuration
 
-1. Open the Administration User interface.
-2. Go to `Settings | General`.
-3. Turn on `Login with SSO`.
-4. Enter the administrator email.
-5. Select the SSO protocol. Choose `Saml 2` or `Open ID Connect`.
+1. Press the `Tools & Settings` then press `General`.
+2. Open the `Security` pane and press `Identity provider`.
```

Comment:
@jasonnovichRunAI maybe we should write "+ identity provider"? which is actually what is written in the button?

```suggestion
2. Open the `Security` pane and press `+Identity provider`.
```

Reply:
```suggestion
2. Open the `Security` pane and press `+Identity provider`.
```